src/common/tiles/MovieTile/index.js

```diff
@@ -2,57 +2,38 @@ import {
   Container,
   Details,
   InfoContent,
-  MovieWrapper,
   Poster,
   StyledRatings,
   Title,
-  Wrapper,
   Year,
 } from "./styled";
-import Mulan from "./poster.jpg";
 import { Genre } from "./Genre";
-import { useDispatch, useSelector } from "react-redux";
-import { useEffect } from "react";
-import {
-  fetchPopularMovies,
-  selectPopularMovies,
-} from "../../../popularMoviesSlice";
 
 import { imageBaseUrl } from "../../../ApiValue";
+import noPoster from "./noPoster.png";
 
-export const MovieTile = () => {
-  const dispatch = useDispatch();
-
-  const movies = useSelector(selectPopularMovies);
-
-  useEffect(() => {
-    dispatch(fetchPopularMovies());
-  }, [dispatch]);
-
+export const MovieTile = ({
+  id,
+  poster_path,
+  title,
+  release_date,
+  vote_average,
+  vote_count,
+}) => {
   return (
-    <Wrapper>
-      {movies.map(
-        ({
-          id,
-          title,
-          poster_path,
-          vote_average,
-          vote_count,
-          release_date,
-        }) => (
-          <Container key={id} id={id}>
-            <Poster src={`${imageBaseUrl}/w342${poster_path}`} alt=""></Poster>
-            <InfoContent>
-              <Details>
-                <Title>{title}</Title>
-                <Year>{release_date}</Year>
-                <Genre />
-              </Details>
-              <StyledRatings rating={vote_average} votes={vote_count} />
-            </InfoContent>
-          </Container>
-        )
-      )}
-    </Wrapper>
+    <Container key={id} id={id}>
+      <Poster
+        src={poster_path ? `${imageBaseUrl}/w342${poster_path}` : noPoster}
+        alt=""
+      ></Poster>
+      <InfoContent>
+        <Details>
+          <Title>{title}</Title>
+          <Year>{release_date}</Year>
+          <Genre />
```

Comment:
Widzę, że Genre masz dalej na sztywno. Tak ma być?

Reply:
będzie pobierało się z API, ale narazie nie wrzucałam tego, bo nie wiem jak to zrobic zeby wyswietlaly sie same nazwy a nie id nazw